ansible_base/lib/abstract_models/user.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib.auth.models import AbstractUser, UserManager
+
+
+class AbstractDABUser(AbstractUser):
```

Comment:
:+1: Let's plan to migrate more fields from downstream back up, and figure more potential points of integration. Shipping with an abstract org/team/user is going to present the most sane behavior for new apps integrating DAB.